### ISSUE TITLE
Bech32 & segwit address support?

### ISSUE THREAD

**Issue by afk11** [bitpay/bitcore-lib#154]
I've been reviewing for common libraries that may not yet have segwit address support. I haven't seen anyone work on it yet, please add this soon!

https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki

**Comment by youssefgh** [comment]
Any news about this ? can't get UTXOs for bech32
`Invalid address: Checksum mismatch. Code:1`

**Comment by afk11** [comment]
*cough* https://github.com/bitpay/bitcore-lib/issues/119

**Comment by youssefgh** [comment]
Yes, @afk11 am already using bitcoinjs. i only use bitpay's insight-api to query balances and UTXOs
I think i should start looking for alts
Thanks for the heads up !

**Comment by coreyphillips** [comment]
@youssefgh Did you ever manage to find an alternative that support Bech32? I've run into the same issue since the insight-api still doesn't support it.

**Comment by youssefgh** [comment]
@coreyphillips yes, a tricky one but it work
am using electrumx server (you can run your own or use one of the public ones) check [my wallet](https://github.com/youssefgh/mooo-wallet) to see how 
since electrumx doesn't support Rest, i had to create a java proxy to convert Rest calls to Json-rpc over TCP calls, check it out [here](https://github.com/youssefgh/tcp-rest-proxy)
You can test that live on [testnet](http://testnet.wallet.mooo.tech) or [mainnet](http://wallet.mooo.tech)

**Comment by coreyphillips** [comment]
@youssefgh That's awesome, thanks. I'll give it a look. Otherwise, I'll end up creating an alternative api to insight-api as well.
